x-pack/plugins/secops/common/graphql/types.ts

```diff
@@ -53,12 +55,35 @@ export interface SourceFields {
   timestamp: string /** The field to use as a timestamp for metrics and logs */;
 }
 
+export interface SuricataEvents {
+  timestamp: string;
+  eventType: string;
+  flowId: string;
+  proto: string;
+  srcIp: string;
+  srcPort: string;
```

Comment:
should `srcPort` and `destPort` be numeric?

Reply:
👍